x-pack/test/security_solution_cypress/cypress/tasks/timeline.ts

```diff
@@ -213,6 +214,10 @@ export const changeTimelineQueryLanguage = (language: 'kuery' | 'lucene') => {
 };
 
 export const addDataProvider = (filter: TimelineFilter): Cypress.Chainable<JQuery<HTMLElement>> => {
+  cy.get(TOGGLE_DATA_PROVIDER_BTN).click();
+  // Cypress doesn't properly wait for the data provider to finish expanding, so we wait for the animation to finish.
+  cy.wait(600);
```

Comment:
I removed this and ran the test 25 times without any issues. I think if you keep the next line `cy.get(TIMELINE_DATA_PROVIDERS_CONTAINER).should('be.visible');` then the wait should not be necessary